rskj-core/src/test/java/co/rsk/peg/ReleaseTransactionBuilderTest.java

```diff
@@ -904,6 +909,112 @@ void test_VerifyTXFeeIsSpentEquallyForBatchedPegouts_three_pegouts() {
+        List<UTXO> utxos = Arrays.asList(
+            new UTXO(mockUTXOHash("utxo"), 0, Coin.COIN, 0, false, fed.getP2SHScript()));
+        // the federation wallet, without flyover support
+        Wallet fedWallet = BridgeUtils.getFederationSpendWallet(
+            Context.getOrCreate(networkParameters),
+            fed,
+            utxos,
+            false,
+            mock(BridgeStorageProvider.class)
+        );
+        // build release transaction builder for current fed
+        when(activations.isActive(ConsensusRule.RSKIP201)).thenReturn(true);
+        ReleaseTransactionBuilder releaseTransactionBuilder = new ReleaseTransactionBuilder(
+            networkParameters,
+            fedWallet,
+            fed.getAddress(),
+            feePerKb,
+            activations
+        );
+
+        // build batch peg-out transaction
+        ReleaseTransactionBuilder.BuildResult result = releaseTransactionBuilder.buildBatchedPegouts(entries);
+
+        // the proposed feePerKb and minPegoutValue are compatible
+        assertEquals(ReleaseTransactionBuilder.Response.SUCCESS, result.getResponseCode());
+    }
+
+    @ParameterizedTest()
+    @MethodSource("providePegoutMinimumParameters")
+    void buildBatchedPegouts_whenUnspendableUXTOs_shouldBuildTransactionSuccessfully(Coin feePerKb, Coin minPegoutValue) {
+        // build federation with 9 members
+        Integer[] memberPKs = new Integer[]{ 100, 200, 300, 400, 500, 600, 700, 800, 900 };
+        Federation fed = FederationTestUtils.getFederation(memberPKs);
+        // list of peg-out requests with the given minimum peg-out value
+        List<ReleaseRequestQueue.Entry> entries = Arrays.asList(
+            createTestEntry(1000, minPegoutValue));
+        // this is a bit hacky, but this is to ensure the minPegoutValue chosen
+        // is not dust with the current wallet implementation. Either way this would be
+        // checked later on when building the transaction, but I think is good to have this explicitly
+        BtcTransaction tx = new BtcTransaction(networkParameters);
+        tx.addOutput(entries.get(0).getAmount(), entries.get(0).getDestination());
+        assertFalse(tx.getOutput(0).isDust());
+        // now we need to calculate the min value that an UTXO can use to be spent. This would be the least
+        // amount needed by the change ouput back to the federation address. This could also be the mininum 
+        // pegin value, but since a change output is also an UTXO that the current wallet implementation can use
+        // then we will find and use that value.
+        Coin minSpendableUTXOValue = Coin.SATOSHI;
+        tx.clearOutputs();
+        tx.addOutput(minSpendableUTXOValue, fed.getP2SHScript());
+        while (tx.getOutput(0).isDust()) {
+            minSpendableUTXOValue = minSpendableUTXOValue.add(Coin.SATOSHI);
+            tx.clearOutputs();
+            tx.addOutput(minSpendableUTXOValue, fed.getP2SHScript());
+        }
```

Comment:
Interesting enough this value turns out to be `2700 SATS`. Which is exactly the lowest value UTXO I was able to find in the Bridge address. Meaning given the current wallet implementation, this is the lowest value that the Bridge will not consider it as dust and will proceed to spend it if so desired by the selector UTXO algorithm. 

Here is the tx: https://mempool.space/tx/ab94ce7abfe178025b593ce0f0c7468a984087ae7977800b62cf47b4284d00dd

Reply:
So this is the lowest change output the Bridge will create, right? If a UTXO is used that generates a smaller change, it will remove sats from the user output until it completes 2,700 sats as the change. So we could say that the max it will drain from a user output is 2,700 sats in the case that the change was 0.

And this does not depend on the feePerKb value set in the Bridge, is that right?

Reply:
> So this is the lowest change output the Bridge will create, right? If a UTXO is used that generates a smaller change, it will remove sats from the user output until it completes 2,700 sats as the change. So we could say that the max it will drain from a user output is 2,700 sats in the case that the change was 0.

That is right.

> And this does not depend on the feePerKb value set in the Bridge, is that right?

That is right, reason being because it always uses a [hardcoded feePerKb value of 15,000 SATS](https://github.com/rsksmart/bitcoinj-thin/blob/master/src/main/java/co/rsk/bitcoinj/core/TransactionOutput.java#L246)